web/pdf_viewer.js

```diff
@@ -1610,6 +1610,43 @@ class PDFViewer {
     };
   }
 
+  switchToEditAnnotationMode() {
```

Comment:
Could we make this method private?

Also, can you please move this method slightly such that the `_updateLocation` and `update` methods are still close together (since the former is a helper for the latter)?